platformio.ini

```diff
@@ -100,3 +100,21 @@ lib_deps =
 monitor_speed = 115200
 extra_scripts = 
 	${env.extra_scripts}
+
+; Build settings for the Arduino Nano
+[env:nano]
+platform = atmelavr
+board = nanoatmega328
+framework = arduino
+build_flags = 
+	${env.build_flags}
+	-I./_Boards/Atmel/Board_Nano
+build_src_filter = 
+	${env.build_src_filter}
+	+<../_Boards/Atmel>
+lib_deps = 
+	${env.lib_deps}
+	arduino-libraries/Servo @ 1.1.8
```

Comment:
The Servo library is already listed in the standard environment `[env]` and shouldn't be listed here.
Once the Raspeberry Pico (or other HW) gets supported the library must indeed moved to this location as the lib depends on the architecture.